### PR TITLE
bowerphp can now handle the .bowerrc file put into the parents directories

### PR DESCRIPTION
**Pull request by guillaumeperes**
Like the bowerJS does, this PHP implementation can now get the config values from a bowerrc file placed into the current directory or the parent directories.

### CODE REVIEW

File: src/Bowerphp/Config/Config.php [hunk 1]
```diff
@@ -250,4 +265,16 @@ protected function getHomeDir()
 
         return rtrim($home, '/');
     }
+
+    protected function getBowerrcPath()
```
Comment (garak) [inline src/Bowerphp/Config/Config.php]: Can you please do the following?
* add a phpdoc block
* avoid do/while
* fix indent

**Comment by guillaumeperes** [comment]
Yeah sure! I'll do it within a few days. Pretty busy right now. :-)

File: src/Bowerphp/Config/Config.php (hunk 1 above)
Comment (guillaumeperes) [inline src/Bowerphp/Config/Config.php]: Yeah sure! I'll do it within a few days. Pretty busy right now. :-)

**Comment by garak** [comment]
Still working on this? Otherwise I'll close it

**Comment by garak** [comment]
Dear contributor,
after a few years of inactivity, this project will be archived.
It serves no purpose anymore, and it's time to move forward.
Thanks for your contribution anyway.

**Pull request closed without merging**